### PR TITLE
[FGTS-204] Normalize audio levels

### PR DESCRIPTION
**Pull request by charasyn**
Unfortunately, some files still needed manual adjusting. The automatic
tool that I used didn't work on the sound effect files, probably because
they're too short, so I also manually adjusted the volumes on the
AudioPlayer node. Going from one level to the next, they all sound the
same volume to me so I count this as a success :)

### CODE REVIEW

File: godotproject/Menus/HighScoreScreen/HighScoreScreen.tscn [hunk 1]
```diff
@@ -184,13 +184,13 @@ tracks/0/enabled = true
 tracks/0/keys = {
 "clips": [ {
 "end_offset": 0.0,
-"start_offset": 0.0,
+"start_offset": 0.3,
 "stream": ExtResource( 7 )
 } ],
-"times": PoolRealArray( -0.3 )
+"times": PoolRealArray( 0 )
 }
```
Comment (charasyn) [inline godotproject/Menus/HighScoreScreen/HighScoreScreen.tscn:176]: This fixes the bug with the victory sound not playing in the final level.

File: godotproject/Menus/HighScoreScreen/HighScoreScreen.gd [hunk 2]
```diff
@@ -71,11 +71,8 @@ func _step_thonk_animation():
 		$ThonkSmallSFX.play()
 
 func _play_final_sound():
-	if final_idx >= num_scores_saved:
-		# If we didn't get a high score, play a sad sound :(
-		$SadSFX.play()
```
Comment (fheysel) [inline godotproject/Menus/HighScoreScreen/HighScoreScreen.gd:76]: Did you delete the .wav file that this audiostream player plays?

**Review by fheysel — APPROVED** [review]
One small comment, I cant see anywhere if the sad sfx file was deleted, if so it's good to merge!

File: godotproject/Menus/HighScoreScreen/HighScoreScreen.gd (hunk 2 above)
Comment (charasyn) [reply godotproject/Menus/HighScoreScreen/HighScoreScreen.gd:76]: Yep! "no_high_score.wav" was removed 👍

**Pull request merged**